moveit_ros/moveit_servo/src/utils/command.cpp

```diff
@@ -94,24 +94,31 @@ JointDeltaResult jointDeltaFromJointJog(const JointJogCommand& command, const mo
 
   velocities.setZero();
   bool names_valid = true;
-
-  for (size_t i = 0; i < command.names.size(); ++i)
+  bool command_valid = true;
+  if (command.velocities.size() == command.names.size())
```

Comment:
Might be better to avoid nesting to check whether the names/velocities are different size first, and return early if that's the case